src/main/resources/services/graphql/schema/query.js

```diff
@@ -97,7 +97,7 @@ module.exports = schemaGenerator.createObjectType({
                 var types = env.args.types;
                 var query = env.args.query;
                 var itemIds = env.args.itemIds;
-                var count = env.args.count || Number.MAX_SAFE_INTEGER;
+                var count = env.args.count !== 0 ? (env.args.count || Number.MAX_SAFE_INTEGER) : 0;
```

Comment:
Number.MAX_SAFE_INTEGER is still dangerous as default (dispite its name)
0 as default is safer, as it won't allow to similar performance bugs to sneak in